src/htmlContent/copy-paste-dialog.html

```diff
@@ -0,0 +1,51 @@
+<div id="myModal" class="modal">
```

Comment:
Can you put a screenshot of this in your PR so @flukeout can review the UI as well?